fedimint-testing/Cargo.toml

```diff
@@ -23,21 +23,22 @@ async-stream = "0.3.5"
 async-trait = { workspace = true }
 bitcoin = { workspace = true }
 bitcoincore-rpc = "0.17.0"
-clap = { version = "4.5.7", features = ["derive", "std", "help", "usage", "error-context", "suggestions" ], default-features = false }
+clap = { version = "4.5.7", features = ["derive", "std", "help", "usage", "error-context", "suggestions"], default-features = false }
 fedimint-core = { workspace = true }
 fedimint-api-client = { workspace = true }
-fedimint-client  = { version = "=0.4.0-alpha", path = "../fedimint-client" }
-fedimint-server  = { version = "=0.4.0-alpha", path = "../fedimint-server" }
+fedimint-client = { version = "=0.4.0-alpha", path = "../fedimint-client" }
+fedimint-server = { version = "=0.4.0-alpha", path = "../fedimint-server" }
 fedimint-bitcoind = { version = "=0.4.0-alpha", path = "../fedimint-bitcoind" }
 fedimint-logging = { workspace = true }
 fedimint-rocksdb = { version = "=0.4.0-alpha", path = "../fedimint-rocksdb" }
+fedimint-ln-common = { version = "=0.4.0-alpha", path = "../modules/fedimint-ln-common" }
```

Comment:
I thought we had a check that prevented adding module specific crates into `fedimint-testing`